ts-lint-rules/tslint.angular.json

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
Because we have a job **"dist-copy:tslint"** and we ship tslint.json with our package, which is actually now extends from a non-existing file. I think we should fix that first if we still want to deliver tslint rules to our consumers, someone can still be using those

Reply:
@dmytrofirsov, is there a ticket for figuring this out?